src/depots/depot.schema.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Given that we have a schema, should there maybe be a test case that validates all depot JSONS are compatible with the schema?

Reply:
I'll raise an issue to resolve this